cid-redirects.json

```diff
@@ -1498,7 +1484,6 @@
   "/cid/10192": "/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/rapid7-source",
   "/cid/10195": "/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-bigquery-source",
   "/cid/10196": "/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/zero-networks-segment-source",
-  "/cid/10213": "/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/trellix-mvisio-epo-source",
```

Comment:
good catch! looks like the same cid is used twice for different sources. I'll create a separate ticket for this.